tools/import-image-tests

```diff
@@ -49,6 +50,10 @@ def load_test(name, fullpath) -> dict:
     if not req:
         raise ValueError("no compose-request")
 
+    distros, archs = list(map(list, zip(*filters)))
```

Comment:
I believe just `distros, archs = zip(*filters))` is enough, no need to turn things into intermediate lists.